examples/ipv4.rs

```diff
@@ -0,0 +1,120 @@
+use packetvisor::pv;
+use signal_hook::SigId;
+use std::{
+    env,
+    io::Error,
+    sync::atomic::{AtomicBool, Ordering},
+    sync::Arc,
+};
+
+fn main() {
+    let args: Vec<String> = env::args().collect();
+    if args.len() != 2 {
+        println!("./ipv4 <interface name>");
+        std::process::exit(-1);
+    }
+
+    let mut if_name = String::new();
+    let chunk_size: u32 = 2048;
+    let chunk_count: u32 = 1024;
+    let rx_ring_size: u32 = 64;
+    let tx_ring_size: u32 = 64;
+    let filling_ring_size: u32 = 64;
+    let completion_ring_size: u32 = 64;
+
+    for i in 1..args.len() {
+        match i {
+            1 => {
+                if_name = args[1].clone();
+            }
+            _ => {
+                panic!("abnormal index");
+            }
+        }
+    }
```

Comment:
How about removing these codes and changing line 17?
lines from 11 to 15 already checks if arguments are inserted right or not, so that `match` is not needed here.
line 17 could be changed like this:
`let if_name: String = args[1].clone();`